cxbox-api/src/main/java/org/cxbox/api/data/dto/DataResponseDTO.java

```diff
@@ -99,4 +124,25 @@ public void transformData(Function<DataResponseDTO, DataResponseDTO> function) {
 		function.apply(this);
 	}
 
+	@Getter
+	@Setter
+	@SuperBuilder
+	public static class CnangedNowParam {
+
+		/**
+		 * Set of field names that are currently being modified.
+		 * <p>
+		 * Contains only the names of changed fields.
+		 */
+		private Set<String>  changedNow;
```

Comment:
add comment why we need this - why private DataResponseDTO changedNowDTO -> changedFields fields is not enougth